src/main/scala/co/teapot/tempest/graph/DirectedGraph.scala

```diff
@@ -44,6 +48,11 @@ trait DirectedGraph {
     * */
   def inNeighbors(id: Int): IndexedSeq[Int]
 
+  def neighbors(id: Int, direction: EdgeDir): Int = direction match {
+    case EdgeDirOut => outDegree(id)
```

Comment:
shouldn't this be
case EdgeDirOut => outNeighbors(id)
case EdgeDirIn => inNeighbors(id)